adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -303,8 +303,7 @@ def connect(self, clean_session=True):
                 raise MMQTTException("Invalid broker address defined.", e)
 
         # Fixed Header
-        fixed_header = bytearray()
-        fixed_header.append(0x10)
+        fixed_header = bytearray([0x10])
```

Comment:
Could you also do the same to the fixed header packet within `publish`: https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/master/adafruit_minimqtt/adafruit_minimqtt.py#L464? Thanks!

Reply:
Again, good eye! I'd like to note that the `MQTT_PUB` const is already a bytearray & only used once throughout `adafruit_minimqtt.py`. Now my initial reaction is to get rid of the (rather non-descriptive) const and change the line on L464 to something like `pub_hdr_fixed = bytearray([0x30 | retain | qos << 1])`. This would be less computationally intensive as opposed to `pub_hdr_fixed = bytearray([MQTT_PUB[0] | retain | qos << 1])`. If you approve the first solution, then I'll go with the first solution. Otherwise I'll just commit the second solution.

PS not sure what the comment `# [3.3.1.2], [3.3.1.3]` is about on L466. I'm assuming its a reference to some revision of a MQTT standard that I'm unaware of.

Reply:
reading the source further, I think I can consolidate the [lines 468-470](https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/blob/1adb75d1f247e740df07b62d40a23e0dfdc6cb5d/adafruit_minimqtt/adafruit_minimqtt.py#L468-L470) to  `pub_hdr_var = struct.pack('>H', len(topic))` seen as how `struct` is already being imported.

Reply:
> PS not sure what the comment # [3.3.1.2], [3.3.1.3] is about on L466. I'm assuming its a reference to some revision of a MQTT standard that I'm unaware of.

http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/os/mqtt-v3.1.1-os.html#_Toc398718039

>  Now my initial reaction is to get rid of the const and change the line on L464 to something like pub_hdr_fixed = bytearray([0x30 | retain | qos << 1])

I like this solution, let's stick with it.

Reply:
thanks for the reply. I've commit the first solution. Also, thanks for the link to MQTT specifications (primary reason I like to contribute is for opportunities to learn).